system/gatekeeper/templates/constraint-prometheusrule-alert-labels.yaml

```
@@ -31,6 +31,14 @@ spec:
           msg := sprintf("rule %s in group %s does not have a valid value for labels.severity (\"critical\", \"warning\" or \"info\")", [alert.rule.alert, alert.group_name])
         }
 
+        violation[{"msg": msg}] {
+          alert := alert_rule[_]
+          # warn if we don't have a labels.playbook when labels.severity == "critical"
+          object.get(alert.rule.labels, "severity", "") == "critical"
+          object.get(alert.rule.labels, "playbook", "") == ""
+          msg := sprintf("rule %s in group %s does not have a valid value for labels.playbook", [alert.rule.alert, alert.group_name])
```

This message should indicate that the severity is the cause for our complaint:
```suggestion
          msg := sprintf("rule %s in group %s does not have a valid value for labels.playbook (required for critical alerts)", [alert.rule.alert, alert.group_name])
```